isis/src/control/objs/BundleUtilities/BundleObservationSolveSettings.cpp

```diff
@@ -240,25 +240,6 @@ namespace Isis {
   }
 
 
-//  BundleObservationSolveSettings::BundleObservationSolveSettings(const BundleObservationSolveSettings &other)
```

Comment:
Was this commented-out code that existed prior to this project or is this newly removed?